mtags/src/main/scala-3/scala/meta/internal/pc/CompletionProvider.scala

```diff
@@ -298,48 +319,59 @@ class CompletionProvider(
           }
         )
 
+      def compareByApplyParams(o1: CompletionValue, o2: CompletionValue): Int =
```

Comment:
Why the name? It seems a bit misleading. Maybe:
```suggestion
      def compareByCompletionType(o1: CompletionValue, o2: CompletionValue): Int =
```

Reply:
Actually, this step was introduced only for proper sort in apply.
It isn't clear but it kind of: if there are NamedArg in completions then we are in Apply  and it's needed to put NamedArg between `Compiler` and `Scope`+`Workspace` items.

Reply:
`comparyInApplyParams` then? `compareByApplyParams` seems like we are comparing the apply tree for each completion.